js/news.js

```diff
@@ -183,54 +183,66 @@ function loadNewsGrid() {
                 iTop = $("#LoadPageNews").children('div').eq(id - 1).offset().top - $("#highlights").offset().top - 65 - 34; // 34 for date of news
                 $(sScrollElement).animate({ scrollTop: iTop }, 500);
             }
-            initHighlightCarousal();
+            NewsSliderConfig()
         },
         complete: function () {
             oNewsContainer.removeClass(sLoadingClass);
         }
     });
 }
 
-function initHighlightCarousal() {
-    $('.item4-carousel').owlCarousel({
-        autoPlay: 2500,
-        autoplay: 2500,
-        slideSpeed: 800,
-        slidespeed: 800,
-        autoplaySpeed: 800,
-        navSpeed: 800,
-        paginationSpeed: 800,
-        stopOnHover: true,
-        items: 4,
-        rewind: true,
-        loop: true,
-        itemsDesktop: [1170, 3],
-        itemsDesktopSmall: [1024, 2],
-        itemsTabletSmall: [768, 1],
-        itemsMobile: [480, 1],
-        pagination: false,  // Hide pagination buttons
-        navigation: true,  // Show next and prev buttons
-        nav: true,  // Show next and prev buttons
-        navigationText: ["<i class='fa fa-angle-left'></i>", "<i class='fa fa-angle-right'></i>"],
-        navText: ["<i class='fa fa-angle-left'></i>", "<i class='fa fa-angle-right'></i>"],
-        dots: false
-        ,responsive: {
-            0: {
-                items: 1
-            }
-            ,
-            570: {
-                items: 2
-            }
-            ,
-            855: {
-                items: 3
-            }
-            , 1140: {
-                items: 4
-            }
-        }
-    });
+function NewsSliderConfig() {
+    $.getJSON("/Configurations/NewsSlider.json", function (data) {
+        initHighlightCarousal(data);
+    })
+};
+
+function initHighlightCarousal(nSliderConfig) {
+    if (typeof nSliderConfig !== 'null' || typeof nSliderConfig !== 'undefined' || typeof nSliderConfig !== "" || typeof nSliderConfig !== 'false') {

```

Comment:
kindly check the syntax for typeof undefined and null check.

Reply:
Updated the syntax